### ISSUE TITLE
Hyper parameters in finetune

### ISSUE THREAD

**Issue by ToheartZhang** [ROGERDJQ/RoBERTaABSA#1]
Thanks for your great work. Notice that the hyperparameter settings in the source code and the paper are different in finetuning. Which setting can reproduce the results of the paper?

```python
parser.add_argument("--lr", type=float, default=2e-5

#######hyper
n_epochs = 40
pool = "max"
smooth_eps = 0.0
dropout = 0.5
#######hype
```

Besides, how did you get your test set? Thanks for your help.

**Comment by ROGERDJQ** [comment]
Thanks for this issue. I just notice that some hyperparameter settings in the paper, especially the learning rate, are written wrongly by mistake. Please use the setting in the code. And I will give a detailed description of the experiment settings later if something still has the mistake.

**Comment by ToheartZhang** [comment]
Thanks for your reply. Besides, which roberta model did you choose to achieve the result listed in the paper? (roberta-base/large, xlm-roberta-base/large)

**Comment by ROGERDJQ** [comment]
Both are in base models.  FYI, you can find the test data in Semeval 2014.

**Comment by ToheartZhang** [comment]
Thanks for your patient reply! There is still a huge gap between your results and my roberta-base model's. I'll try to figure it out.

**Comment by ROGERDJQ** [comment]
Then it would be great if you could share your results on RoBERTa-base. Maybe we could find out the reasons.
